src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -992,18 +992,20 @@ def _get_auth_provider_latest_api_version(cli_ctx):
     return api_version
 
 
-def _update_provider(cli_ctx, namespace, registering, wait, accept_terms=None):
+def _update_provider(cli_ctx, namespace, registering, wait, mgID=None, accept_terms=None):
```

Comment:
Hump case is not recommended for Python variable names. Please refer to [Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
> Variable names follow the same convention as function names.
> mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility.